### PR TITLE
Add SkipBody and SkipResBody functions to Config. it used to skip record "body" and "resBody" fields when write log.

### PR DESCRIPTION
**Pull request by piyongcai**
* Add SkipBody and SkipResBody functions to Config. it used to skip record "body" and "resBody" fields when write log.

// SkipBody defines a function to skip log  "body" field when returned true.
//
// Optional. Default: nil
SkipBody func(c *fiber.Ctx) bool

// SkipResBody defines a function to skip log  "resBody" field when returned true.
//
// Optional. Default: nil
SkipResBody func(c *fiber.Ctx) bool

### CODE REVIEW

**Review by ReneWerner87 — CHANGES_REQUESTED** [review]
please add tests that test the new parts that have been added

File: fiberzap/config.go [hunk 1]
```diff
@@ -12,6 +12,16 @@ type Config struct {
 	// Optional. Default: nil
 	Next func(c *fiber.Ctx) bool
 
+	// SkipBody defines a function to skip log  "body" field when returned true.
+	//
+	// Optional. Default: nil
+	SkipBody func(c *fiber.Ctx) bool
+
+	// Next dSkipBody defines a function to skip log  "resBody" field when returned true.
```
Comment (ReneWerner87) [inline fiberzap/config.go:20]: "dSkipBody" -> SkipBody

**Comment by piyongcai** [comment]
> please add tests that test the new parts that have been added

Add Test for SkipBody and SkipResBody config item.

File: fiberzap/config.go (hunk 1 above)
Comment (piyongcai) [reply fiberzap/config.go]: fixed comment "dSkipBody" -> SkipBody

**Comment by ReneWerner87** [comment]
@piyongcai sorry for my late reply

would merge the pull request, please only expand the readme in the fiber zap folder with the new possibilities of configuration

**Comment by ReneWerner87** [comment]
@piyongcai

**Review by ReneWerner87 — APPROVED** [review]

**Comment by piyongcai** [comment]
> @piyongcai

sorry，i missing this pr.

**Comment by piyongcai** [comment]
@ReneWerner87 
sorry, i missing this pr,  because some reason....

**Pull request merged**